.devcontainer/postCreateCommand.sh

```diff
@@ -2,15 +2,21 @@
 
 echo "Install Poetry and Python dependencies"
 curl -sSL https://install.python-poetry.org | python -
-poetry install
+poetry config virtualenvs.in-project true
+poetry install --with dev
 poetry run python scripts/fetch_core.py
 poetry run python scripts/zip_templates.py
+poetry run playwright install-deps
+poetry run playwright install
+
+echo "Update apt"
```

Comment:
For this and the rest of the file: if we switch to the docker image, this is unnecessary (and would be faster)